bolt-anything.bash

```diff
@@ -0,0 +1,149 @@
+#!/bin/bash
+
+## STAGE 1 = build llvm-bolt
+## STAGE 2 = Instrument binary to run a workload with it to gather profiles for optimizing
+## STAGE 3 = Merging the created data to one file and remove the not needed data## Actually bug in llvm-bolt https://github.com/llvm/llvm-project/issues/56209
+## STAGE 4 = Optimize the binary with the created profile
+STAGE=
+
+## File or binary you want to instrument and then bolt
+${BINARY:=libLLVM-14.so}
```

Comment:
for the "trick" to work it needs to be `: ${BINARY:=libLLVM-14.so}`, the `:` is an empty statement that does nothing.

https://github.com/koalaman/shellcheck/wiki/SC2223